tests/integration/test_data/gomod_packages.yaml

```diff
@@ -202,3 +202,86 @@ vendored_without_flag:
   repo: https://github.com/cachito-testing/gomod-vendored.git
   ref:  ff1960095dd158d3d2a4f31d15b244c24930248b
   pkg_managers: ["gomod"]
+implicit_gomod:
```

Comment:
Can you please add a comment about the test data?

Reply:
done :)